script/test.py

```diff
@@ -10,6 +10,13 @@
 from lib.util import electron_gyp, execute_stdout, rm_rf
 
 
+if sys.platform == 'linux2':
+    try:
+        import lib.dbus_mock
```

Comment:
Can you please add a comment why it is needed, and why it is ok if it fails?

Reply:
:+1: